README.md

```diff
@@ -50,6 +50,19 @@ ZCBOR_STATE_E(encode_state, n, payload, payload_len, 0);
 
 The CBOR libraries assume little-endianness by default, but you can define ZCBOR_BIG_ENDIAN to change this.
 
+Configuration
+-------------
+
+The C library has a few compile-time configuration options which can be enabled by adding them as compile definitions to the build.
```

Comment:
Would `These compile-time configuration options can be enabled by adding...` work better?

Reply:
Agree, done.